core/tests/puppeteer/lighthouse_setup.js

```diff
@@ -287,6 +288,9 @@ const main = async function() {
   await getCollectionEditorUrl(browser, page);
   await getTopicEditorUrl(browser, page);
   await getSkillEditorUrl(browser, page);
+  if (storyEditorUrl.includes('story_editor')) {
```

Comment:
Why this change? Perhaps add a comment to explain.

Reply:
I switched to a different method so this outdated